helix-rest/src/test/java/org/apache/helix/rest/server/TestPerInstanceAccessor.java

```diff
@@ -463,5 +464,53 @@ public void testValidateWeightForInstance()
     // Must have the results saying they are all valid (true) because capacity keys are set
     // in ClusterConfig
     node.iterator().forEachRemaining(child -> Assert.assertTrue(child.getBooleanValue()));
+    System.out.println("End test :" + TestHelper.getTestMethodName());
+  }
+
+  @Test(dependsOnMethods = "testValidateWeightForInstance")
```

Comment:
If I understand correctly, we are testing changing instance config domain field via rest. Here, the validation of config is actually executed from rest server side, right?

Can we add a small comment about the purpose of this test?

Reply:
This is correct. I will update.

Reply:
Updated.